src/coreclr/jit/lsra.cpp

```diff
@@ -1922,6 +1922,9 @@ void           LinearScan::identifyCandidates()
                             fieldVarDsc->lvLRACandidate                = 0;
                             localVarIntervals[fieldVarDsc->lvVarIndex] = nullptr;
                             VarSetOps::RemoveElemD(compiler, registerCandidateVars, fieldVarDsc->lvVarIndex);
+#if FEATURE_PARTIAL_SIMD_CALLEE_SAVE
+                            VarSetOps::RemoveElemD(compiler, largeVectorVars, fieldVarDsc->lvVarIndex);
```

Comment:
Not sure if it makes sense to check if `fieldVarDsc->lvVarIndex` was part of `largeVectorVars` to begin with.

Reply:
Hmm, I think just clearing it unconditionally should be ok.